.github/workflows/lint.yml

```
@@ -41,3 +41,28 @@ jobs:
         with:
           command:      fmt
           args:         --all -- --check
+  check-spellcheck:
+    name:               Check For Spelling and/or Grammar Mistakes
+    runs-on:            ubuntu-latest
+    env:
+      RUST_BACKTRACE:   full
+    steps:
+      - name:           Cancel Previous Runs
+        uses:           styfle/cancel-workflow-action@0.4.1
+        with:
+          access_token: ${{ github.token }}
+      
+      - name:           Checkout sources & submodules
+        uses:           actions/checkout@master
+        with:
+          fetch-depth:  5
+          submodules:   recursive
+      
+      - name:           Add cargo-spellcheck
+        run:            cargo install cargo-spellcheck
+      
+      - name:           Run spellcheck
+        uses:           actions-rs/cargo@master
+        with:
+          command:      spellcheck
+          args:         check  -m 1 -vv $(find modules/currency-exchange -name *.rs)
```

```suggestion
          args:         check -m 1 -vv $(find modules/currency-exchange -name *.rs)
```

One nitpick (and I think it was even introduced by me in the other PR 😅 )